using-valtimo/plugin/documenten-api/configure-documenten-api-plugin.md

```diff
@@ -39,6 +39,10 @@ Document. A reference to the document has to be available as a process variable.
 document, the resulting document URL will be saved in a process variable.
 
 When creating a process link the following properties have to be entered:
+- **File name** The name of the physical file in which the content of the document is captured, including extension.
```

Comment:
Missing a period after each of the bold texts.

Reply:
What do you mean? Should it be 
```
**File name**. The name of the physical....
```?

The way it is now is in line with the other plugin pages though.

Reply:
I added a . to all pages

Reply:
Honestly, i don't really like the way it looks:

<img width="763" alt="image" src="https://user-images.githubusercontent.com/74657121/191238966-c7b307a3-e9ea-4a34-929a-874995321c15.png">

Reply:
I like it more than without the dot.

Reply:
I dont mind either way